tests/snapshot_tests/test_snapshots.py

```diff
@@ -92,6 +92,10 @@ def test_header_render(snap_compare):
     assert snap_compare(WIDGET_EXAMPLES_DIR / "header.py")
 
 
+def test_list_view(snap_compare):
```

Comment:
I don't think that would add much since we can't assert that events are fired. I think we spoke in the past about how it would be nice to assert that, for example, when I press Enter, a ListView.Selected event is fired.